unity-environment/Assets/ML-Agents/Examples/Walker/Scripts/WalkerGroundContact.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class WalkerGroundContact : MonoBehaviour
```

Comment:
Please add class-level description for this object. (What it does, when it should be used and any assumptions.)

Reply:
Yea, and we could reuse this in the crawler.

Reply:
Just changed it so that it is generic and re-usable, as well as commented.